src/apps/search-result/search-result.test.ts

```diff
@@ -0,0 +1,127 @@
+const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
+
+describe("Search Result", () => {
+  it("Should render the search results as expected", () => {
+    cy.visit(
+      "/iframe.html?id=apps-search-result--search-result&args=pageSizeDesktop:2;pageSizeMobile:2"
+    );
+
+    cy.log("Check length of search result list:");
+    cy.get(".search-result-page__list").find("li").should("have.length", 2);
+
+    cy.log("Do the search results have images?");
+    cy.get(".search-result-page__list .search-result-item img")
+      .should("have.attr", "src")
+      .and("match", coverUrlPattern);
+
+    cy.log("Does the search result have favourite buttons?");
+    cy.get(
+      ".search-result-page__list .search-result-item .button-favourite"
+    ).should("have.attr", "aria-label", "Add to favorites");
+
+    cy.log("Does the search result have a series line?");
+    cy.get(
+      ".search-result-page__list .search-result-item .horizontal-term-line"
+    ).should(
+      "contain.text",
+      "Nr. 1  i serienDummy Some SeriesNr. 1  i serienDummy Some Series"
```

Comment:
Consider adding a comment explaining why this text is somewhat odd.

An alternative approach would be to modify the mocked response to return something that is closer to what we expected.

Reply:
Yes UI agree. I think we need to revise the text when we get some meaningful data from DBC. I'll write a comment about that.